lib/rides/pages/ride_detail_page.dart

```diff
@@ -189,7 +189,10 @@ class _RideDetailPageState extends State<RideDetailPage> {
           onPressed: _showWithdrawDialog,
         );
       default:
```

Comment:
would probably be better to make this case explicit (which statuses apply?)

Reply:
.cancelledbyDriver, .cancelledbyRider and .rejected

Reply:
yeah so I would put those cases explicitly in the code as case statements